### ISSUE TITLE
FIFA 17 (BLUS31593) Stuck on Title Screen after updating to version 1.08

### ISSUE THREAD

**Issue by MakiSalonga** [RPCS3/rpcs3#12238]
## Details

I was able to run FIFA 17 on my laptop smoothly without errors, but once I installed the latest game update, it freezes at the title screen, then fades to black.

#### 1. Please attach RPCS3's log.
 [RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/8926045/RPCS3.log.gz)

#### 3. Please attach screenshots of your problem.
![2022-06-17 (1)](https://user-images.githubusercontent.com/60142215/174263046-6319cad6-7470-4663-a770-642998273904.png)
![2022-06-17](https://user-images.githubusercontent.com/60142215/174263092-44eae909-a6a1-4284-b278-7a22a639153b.png)

#### 5. Please provide your system configuration:
11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz | 8 Threads | 11.77 GiB RAM | TSC: 2.419GHz | AVX-512+ | FMA3
Operating system: Windows, Major: 10, Minor: 0, Build: 19044, Service Pack: none, Compatibility mode: 0

#### Please include.
* Anything else you deem to be important

**Comment by Jonathan44062** [comment]
FIFA Games shouldun't be updated, known issue, removing the update fixes the problem
https://wiki.rpcs3.net/index.php?title=Help:Installing_Game_Updates#Removing_game_updates/game_data

**Comment by Satan86** [comment]
Yes, but it should still work even when updated, it's a Valid Issue

**Comment by benderscruffy** [comment]
RPCS3 v0.0.26-14720-bbd308a9 Alpha | master
still an issue
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/10765270/RPCS3.log.gz)

**Comment by FlexBy420** [comment]
Is this still an issue?

**Comment by MakiSalonga** [comment]
I don't think so.

**Comment by digant73** [comment]
@MakiSalonga is that still an issue?

**Comment by MakiSalonga** [comment]
@digant73 not anymore

**Comment by digant73** [comment]
thanks. If the issue is no longer present please close this issue
